test/action_controller/serialization_test.rb

```diff
@@ -225,11 +222,16 @@ def test_render_with_cache_enable
           }
         }
 
-        assert_equal 'application/json', @response.content_type
-        assert_equal expected.to_json, @response.body
+        ActionController::Base.cache_store.clear
+        Timecop.freeze(DateTime.now) do
```

Comment:
You probably mean Time here. I don't think we're running any tests on the pre-Gregorian calendar :) see https://github.com/ruby/ruby/commit/45458752d3f54ac81661d93aa55b8b5805f78138